app/src/main/res/layout/user_registration.xml

```diff
@@ -82,12 +83,34 @@
         android:id="@+id/registration_username"
         android:layout_width="336dp"
         android:layout_height="wrap_content"
-        android:layout_marginTop="64dp"
+        android:layout_marginTop="268dp"
         android:ems="10"
         android:hint="@string/username_capitalU"
         android:inputType="textPersonName"
         app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintHorizontal_bias="0.497"
+        app:layout_constraintHorizontal_bias="0.506"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/habit_smasher_header" />
+
+    <ImageView
+        android:id="@+id/registration_image"
+        android:layout_width="157dp"
+        android:layout_height="160dp"
+        android:layout_marginTop="16dp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/textView2"
```

Comment:
What is `textView2`? Let's rename this to a better name.

- [ ] rename `textView2`